src/locales/en.json

```diff
@@ -149,10 +149,15 @@
       "see_video": "If you'd like to watch a video walkthrough of how to use the calculator, click <1>here</1>.",
       "call_to_action": "Calculate the approximate COVID risk of an activity or relationship"
     },
-    "risk_step_label": "Step 2: Describe the activity or relationship",
-    "type_of_interaction": "Is this a specific activity or an ongoing relationship?",
-    "risk_group_empty_warning": "First, enter your location",
+    "risk_step_label": "Step 2: Describe the scenario",
+    "type_of_interaction": "What do you want to learn?",
+    "one_time_interaction_risk_message": "If one person is infected, there is a baseline <1>{{ percentage }} chance of transmission per hour</1>.",
+    "repeated_interaction_risk_message": "If one person is infected, there is a baseline <1>{{ percentage }} chance of transmission per week</1>.",
+    "partner_interaction_risk_message": "If this person is infected, there is a baseline <1>{{ percentage }} chance of transmission per week</1>.",
+    "whats_next_interaction_risk_message": "How likely is it someone is infected? It depends on your location and the choices of people around you. This calculator will also walk you through <1>choices you can make</1> to change the baseline risk (backed by <4>research</4>!)",
+    "risk_group_empty_warning": "Enter your location first.",
     "saved_scenario_loaded_message": "You selected this scenario: <strong>{{ scenarioName }}</strong>. Do the points below look like the activity you have in mind? If not, adjust it until it looks right.",
```

Comment:
A small request that isn't directly related to this PR, but IMO is fine to include: 

The "do the **points** below" wording is confusing because we use "points" to mean "risk points" in some places in the documentation. Could we think up a different word? (Happy to move this to a separate issue/PR if it isn't an easy/obivous solution.)

Maybe just: replace "points" with "items" or "selections"

Reply:
"answers", maybe?

Reply:
If you go with answers I'd say something like "answers to the questions below". That's clear to me.

Reply:
"If needed, adjust these prefilled answers to match your Grocery store for 60 minutes (average # of shoppers) scenario."?

Reply:
Works for me! Thanks!